tests/ens/test_setup_name.py

```diff
@@ -127,7 +130,7 @@ def test_setup_name_unowned_exception(ens):
 
 def test_setup_name_unauthorized(ens, TEST_ADDRESS):
     with pytest.raises(UnauthorizedError):
-        ens.setup_name('root-owned-tld', TEST_ADDRESS)
+        ens.setup_name('root-owned-tld.eth', TEST_ADDRESS)
```

Comment:
This test was originally attempting to set up a TLD, which would require owning the root. (Hence the name `root-owned-tld`)

Since the address sending the transaction doesn't own the root, it's supposed to fail.